docs/configuration.rst

```diff
@@ -1498,6 +1498,45 @@ Default: False
    The created ``needs.json`` file gets stored in the ``outdir`` of the current builder.
    So if ``html`` is used as builder, the final location is e.g. ``_build/html/needs.json``.
 
+.. _needs_permalink_file:
+
+needs_permalink_file
```

Comment:
Is this option really needed?
I mean, do you see any use cases where a user wants to specify his/her own permalink.html file?

For me, only the used file name in the final `_build/html/ for `permalink.html? to be configurable. (e.g. `sttic_ids.html` instead of permalink)
The file content itself can be always the same.

What do you think?

Reply:
I mainly wanted the path to be configurable. 
In our project the web server does not host the _build/html directory, but the _build directory.
So I would need to change the needs_permalink_file option from the default "/peramlink.html" to "/html/permalink.html".

Reply:
Agreed and changed. See below comment.